packages/block-editor/src/store/defaults.js

```diff
@@ -100,26 +100,31 @@ export const SETTINGS_DEFAULTS = {
 		{
 			name: _x( 'Small', 'font size name' ),
 			size: 13,
+			sizePx: '13px',
```

Comment:
What do you think if we move these to the `defaults.native.js` file? We'd need to move the whole `fontSizes` but I think it'd make sense since this `sizePx` is for native only.

Reply:
Good idea!